src/test-gui/java/de/qabel/desktop/ui/LayoutGuiTest.java

```diff
@@ -20,6 +21,8 @@
 public class LayoutGuiTest extends AbstractGuiTest<LayoutController> {
 
     private QuotaState quotaState;
+    private static final String ABOUT_VIEW = "#aboutViewRoot";
```

Comment:
Stuff like that better resides inside a page object.
Page objects are the layer between the test that should be as clean and human readable as possible and the gui / gui accessors that are filles with strange names and signes like `#` and details nobody wants to know about when reading the test. (kind of what you did with the `showAboutViewOnStart` but with the additional fact that the id of the root node of the AboutView is nothing the LayoutGuiTest should care about. It shouldn't be coupled to that detail.